doc/3/migration/how-to/index.md

```diff
@@ -0,0 +1,163 @@
+---
+code: false
+type: page
+title: SDK Java to SDK Jvm
+description: SDK Java to SDK Jvm
+order: 99
+---
+
+# SDK Jvm
+
+In this tutorial you will learn how to migrate from the Kuzzle **Java SDK** to the **Jvm SDK**.
+
+::: info
+Having trouble? Get in touch with us on [Discord](http://join.discord.kuzzle.io)!
+:::
+
+## Installation
+
+The installation of the SDK is the same as the Java SDK.
+You can find the SDK JARs directly on [bintray](https://bintray.com/kuzzle/maven/sdk-jvm). Download and add it to your classpath.
+
+### Maven:
+**from**
+```xml
+<dependency>
+  <groupId>io.kuzzle</groupId>
+  <artifactId>kuzzle-sdk-java</artifactId>
+  <version>3.0.0</version>
+  <type>pom</type>
+</dependency>
+```
+**to**
+```xml
+<dependency>
+  <groupId>io.kuzzle</groupId>
+  <artifactId>sdk-jvm</artifactId>
+  <version>1.0.0</version>
+  <type>pom</type>
+</dependency>
+```
+
+### Gradle:
+**from**
+```groovy
+implementation 'io.kuzzle:kuzzle-sdk-java:3.0.0'
+```
+**to**
+```groovy
+implementation 'io.kuzzle:sdk-jvm:1.0.0'
+```
+
+### Ivy:
+**from**
+```html
+<dependency org='io.kuzzle' name='kuzzle-sdk-java' rev='3.0.0'>
+  <artifact name='kuzzle-sdk-java' ext='pom' ></artifact>
+</dependency>
+```
+**to**
+```html
+<dependency org='io.kuzzle' name='sdk-jvm' rev='1.0.0'>
+  <artifact name='sdk-jvm' ext='pom' ></artifact>
+</dependency>
+```
+
+# Breaking changes
+
+## Options handling
+
+The main difference between those two SDKs is the way how we handle options.
+
+In the Java SDK, we have an `Options` class for each option type such as [SearchOptions](/sdk/java/3/core-classes/search-options),
+[CreateOptions](/sdk/java/3/core-classes/create-options) or [SubscriptionOptions](/sdk/java/3/core-classes/subscribe-options).
+
+It allowed us to not write as much as overloads as available options.
+If the method signature could have 5 options, we had to have 5 different signatures,
+or one with the options object, and another without it.
+
+Since the new Jvm SDK is written in Kotlin, we now use the optional parameters directly in
+the function signature, and add a `@JvmOverload` annotation to it in order to generate
+as much as overload as we need.
+
+That is why in this SDK, **we do not have any `Options` object**.
```

Comment:
Oh, I didn't know that the new JVM SDK works with overloads. I hope this won't be too cumbersome for users... And it circumvents our specs about how optional arguments should be handled by our SDKs (https://github.com/kuzzleio/meeting-notes/blob/master/specs/2019-06-20%20Optional%20parameters%20in%20SDKs.md)

But, well... I did a quick check. And I found out that the JVM SDK generated this overload for `document:search`:

```
public CompletableFuture<SearchResult> search(
      String index,
      String collection,
      ConcurrentHashMap<String, Object> searchQuery,
      String scroll,
      Integer from) throws NotConnectedException, InternalException
```

This is incorrect, as the `from` cannot be used with the `scroll` option:

```
$ kourou document:search foo bar --scroll="2s" --from 10
 
 🚀 Kourou - Searches for documents
 
 [ℹ] Connecting to ws://localhost:7512 ...
 [X] Error Validation Failed: 1: using [from] is not allowed in a scroll context
```

I think the automated overload generation should be controlled in some way, to exclude some unusable combinations.